khtools/sequence_encodings.py

```diff
@@ -138,17 +244,48 @@
 AMINO_KETO_TRANSLATION = str.maketrans(AMINO_KETO_MAPPING)
 WEAK_STRONG_TRANSLATION = str.maketrans(WEAK_STRONG_MAPPING)
 PURINE_PYRIMIDINE_TRANSLATION = str.maketrans(PURINE_PYRIMIDINE_MAPPING)
+
+PEPTIDE_MAPPINGS = {"hp": HP_MAPPING,
+                    "hydrophobic-polar": HP_MAPPING,
+                    "dayhoff": DAYHOFF_MAPPING,
+                    'dayhoff_v2': DAYHOFF_V2_MAPPING,
+                    'botvinnik': BOTVINNIK_MAPPING,
+                    "aa9": AA9_MAPPING, 'gbmr4': GBMR4_MAPPING,
```

Comment:
just a suggestion for aesthetics. "aa9": AA9_MAPPING, 'gbmr4': GBMR4_MAPPING, can be in 2 separate lines, use single quotes and double quotes consistently